File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
    // ask user for input that positive and less 23
    int n;
    do
    {
        n = get_int("enter a number: ");
    } while (n <= 0 || n >= 23);
    for (int i = 0; i < n; i++) // for loop go down
    {
        for (int j = 0;j < n + 1 ;j++) // for loop go sideways
        {
            if (j < n - i - 1)
            {
            printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
# get input from user
while True:
    n = int(input("height: "))
    if n > 0 and n < 24 :
        break

print()

# print the pyramid
for i in range(n):
    for j in range(n):
        if j < n - i - 1:
            print(" ", end='')
        else :
            print("#", end ='')
    print()